README.md

```diff
@@ -29,7 +29,7 @@ Interpretation of Computer Programs, JavaScript Adaptation
 To build,
 
 ```{.}
-$ git clone https://github.com/source-academy/js-slang.git
+$ git clone --recurse-submodules https://github.com/source-academy/js-slang.git
```

Comment:
Would be good to add instructions on how to update submodules as well, for those who have already cloned the repo,